src/operators/Remap/avtRemapFilter.C

```diff
@@ -678,7 +678,7 @@ avtRemapFilter::CalculateCellVolumes(vtkDataSet* in_ds, const char* name)
     int nCells = in_ds->GetNumberOfCells();
     
     // Setup the volumeArray
-    if (name != "")
+    if (name != std::string(""))
```

Comment:
This is gettting outside my expertise, but what about the case of name being NULL? Also, this seems like it relies on the compiler promoting name to a std:: string. Perhaps something like "name != std::nullptr && *name != '\0'" would be better?

Reply:
Looking over the standard some more, "std::nullptr" should be just "nullptr".

Reply:
Good point ... point on the nullptr. Though OS X does not like std::nullptr. Only nullptr.

I realized Eddie wrote the code so pulling him in to comment review.

Reply:
This if statement should probably be deleted actually. This is a private function that only ever gets called with the name arg provided.

Reply:
I can delete it so it is just 

new:
        volumeArray->SetName(name);

old:
    if (name != nullptr && *name != '\0')
    {
        volumeArray->SetName(name);
    }

As that would certainly remove the original warning.

Reply:
Perfect! Thanks Allen

Reply:
I have removed the check once the tests pass you all can do the merge request.